test/livebook/live_markdown/markdown_helpers_test.exs

```diff
@@ -24,16 +34,56 @@ defmodule Livebook.LiveMarkdown.MarkdownHelpersTest do
       assert markdown == reformat(markdown)
     end
 
+    test "inline code with a line break" do
+      markdown = "The Game, `Mrs\nHudson`, is on!"
+      assert markdown == reformat(markdown)
+    end
+
+    test "inline code with extra spaces" do
+      markdown = "The Game, `Mrs Huds  on`, is on!"
+      assert markdown == reformat(markdown)
```

Comment:
Not sure if this one should pass or not.

`code` by itself doesn't set `white-space: pre`, but GFM does _preserve_ the spaces in the rendered output (so they show up if copy-pasted, for instance).

Reply:
Interesting, I wouldn't expect the content to change, but that's more up to the parser in this case anyway.

Reply:
The parser correctly returns an `EarmarkParser.Block.Para` for this, but the code in `EarmarkParser.Ast.Inline.converter_for_code` then explicitly squashes whitespace